.github/workflows/ci.yaml

```diff
@@ -77,16 +69,16 @@ jobs:
 
       - name: Run headless tests
         id: run-tests
-        run: npm run test -- --watch=false --browsers=ChromeHeadless 
+        run: npm run test -- --watch=false --browsers=ChromeHeadless
 
       - name: Build angular project
         id: build-project
         run: npm run build
 
-      - name: Log in to github container registry  
+      - name: Log in to github container registry
         id: login-registry
-        run: | 
-          echo "${{ secrets.github_token }}" \
+        run: |
+          echo "${{ github.token }}" \
           | docker login ${{ inputs.docker_registry }} -u ${{ github.actor }} --password-stdin
 
       - name: Build docker image
```

Comment:
Let's find out existing actions for docker `build` and `push`

Reply:
Done